app/controllers/games_controller.rb

```diff
@@ -32,19 +34,16 @@ def edit
   # POST /games
   # POST /games.json
   def create
-    @game = Game.new(game_params)
-    @game.current_turn = 0
-    @game.black_player_id = current_player.id if @game.white_player_id == 0
-    
-    respond_to do |format|
+    ActiveRecord::Base.transaction do
```

Comment:
thanks for the explanation, @lasley. your comment makes perfect sense in the context of those bullet points.

my thinking in using a transaction was to introduce the idea of operation atomicity--the idea that in a web app, we frequently have db operations that we want to happen in all-or-nothing fashion. you are 100% correct that we can eliminate the transaction block around the code as-is. 

i was trying to lead matt towards a model-level abstraction that would take care of all the object configuration (and timer creation, etc) required to start a new chess game. you couldn't possibly have known this, so i apologize for the wire-crossing here. i'll be better about explaining higher-level goals to matt as we work through our sessions.